.travis.yml

```diff
@@ -1,44 +1,59 @@
-sudo: false
+dist: xenial
 language: python
+
 python:
-- '2.7'
-- '3.4'
-- '3.5'
-- '3.6'
-- 'pypy-5.4'
+  - '2.7'
+  - '3.4'
+  - '3.5'
+  - '3.6'
+  # - 'pypy2.7-6.0'
+  - 'pypy3.5-6.0'
+
 env:
-- DEPS="pytest~=2.9.0"
-- DEPS="pytest~=3.0.0"
-#- DEPS="pytest~=3.1.0"
+  - DEPS="pytest~=2.9.0"
+  - DEPS="pytest~=3.0.0"
+  #- DEPS="pytest~=3.1.0"
```

Comment:
commented out?

Reply:
Before already, just indented.
Would be changed in the next step, e.g. by testing against pytest 4.2.

Reply:
Oops did not notice that, sorry